data-app-id.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/src/res/CMakeLists.txt b/src/res/CMakeLists.txt
+index 35830a23e..9bd9e1119 100644
+--- a/src/res/CMakeLists.txt
++++ b/src/res/CMakeLists.txt
+@@ -15,13 +15,13 @@ install(FILES ${help_files} DESTINATION ${FWB_INSTALL_DATADIR}/help/en_US)
+ install(DIRECTORY configlets DESTINATION ${FWB_INSTALL_DATADIR})
+ 
+ if (UNIX AND NOT APPLE)
+-    install(FILES fwbuilder.desktop DESTINATION ${CMAKE_INSTALL_DATADIR}/applications)
+-    install(FILES Icons/16x16/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/16x16/apps)
+-    install(FILES Icons/24x24/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/24x24/apps)
+-    install(FILES Icons/32x32/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/32x32/apps)
+-    install(FILES Icons/48x48/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/48x48/apps)
+-    install(FILES Icons/72x72/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/72x72/apps)
+-    install(FILES Icons/128x128/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/128x128/apps)
+-    install(FILES Icons/256x256/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/256x256/apps)
+-    install(FILES Icons/512x512/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/512x512/apps)
++    install(FILES fwbuilder.desktop DESTINATION ${CMAKE_INSTALL_DATADIR}/applications RENAME com.github.fwbuilder.desktop)
++    install(FILES Icons/16x16/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/16x16/apps RENAME com.github.fwbuilder.png)
++    install(FILES Icons/24x24/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/24x24/apps RENAME com.github.fwbuilder.png)
++    install(FILES Icons/32x32/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/32x32/apps RENAME com.github.fwbuilder.png)
++    install(FILES Icons/48x48/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/48x48/apps RENAME com.github.fwbuilder.png)
++    install(FILES Icons/72x72/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/72x72/apps RENAME com.github.fwbuilder.png)
++    install(FILES Icons/128x128/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/128x128/apps RENAME com.github.fwbuilder.png)
++    install(FILES Icons/256x256/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/256x256/apps RENAME com.github.fwbuilder.png)
++    install(FILES Icons/512x512/fwbuilder.png DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/512x512/apps RENAME com.github.fwbuilder.png)
```

Comment:
You can use rename-deskto-file & rename-icon attributes in the json file to automatically rename those files.